docs/rest.md

```diff
@@ -305,6 +298,26 @@ With the previously defined transformer, the following response structure would
 }
 ```
 
+The following transformer could be used to support it:
+
+
+```js
+const link = new RestLink({
+  uri: '/api',
+  responseTransformer: async response => response.json().then(({data}) => data),
+});
+```
+
+Plaintext responses can be handled by manually parsing and converting them to JSON (using the previously described format that Apollo expects):
```

Comment:
```suggestion
Plaintext, or XML, or otherwise-encoded responses can be handled by manually parsing and converting them to JSON (using the previously described format that Apollo expects):
```